UI/window-basic-main.cpp

```diff
@@ -4900,6 +4903,8 @@ void OBSBasic::ReplayBufferStop(int code)
 	if (!outputHandler || !outputHandler->replayBuffer)
 		return;
 
+	replayBufferButton->setChecked(false);
+
```

Comment:
I'd personally remove this blank line to group it with the `replayBufferButton->setText()` below.